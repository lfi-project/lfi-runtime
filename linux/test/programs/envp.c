#include <assert.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int
main(void)
{
    char *var;
    var = getenv("LFI");
    assert(var && strcmp(var, "1") == 0);
    printf("LFI=%s\n", var);
    var = getenv("NOT_EXIST");
    assert(!var);
    var = getenv("USER");
    assert(var && strcmp(var, "sandbox") == 0);
    printf("USER=%s\n", var);
    return 0;
}
