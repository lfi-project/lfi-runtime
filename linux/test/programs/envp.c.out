LFI=1
USER=sandbox
